variables.tf

```diff
@@ -107,15 +101,9 @@ variable "topics" {
   default     = []
 }
 
-variable "kms_encryption_enabled" {
```

Comment:
No need to remove this.

Reply:
Its not used, it was only used to set the policy

Reply:
hmm ok - I recall we added it in the past because it is needed if any resources use it in their count (such as auth policy). It had to be a boolean, or tf would fail I think. Just if ever had such a requirement in the future, we would need to add it back. Its also consistent with other modules.

Reply:
I can add it back but will need to put some sort of ignore for the linter as it was only used in the policy creation.

Reply:
not if you use it in the validation again. Also you will need to use in some logic for the use case where user passes `kms_key_crn` but has `kms_encryption_enabled` set to false

Reply:
actually we should add validation for that use case so it spits out an error saying `kms_encryption_enabled` must be set to true us passing a value for `kms_key_crn` if you want to use KMS encryption